framework/Source/Camera.swift

```diff
@@ -217,8 +217,8 @@ public class Camera: NSObject, ImageSource, AVCaptureVideoDataOutputSampleBuffer
                         outputWidth = bufferWidth
                         outputHeight = bufferHeight
                     }
-                    let outputTexture = Texture(device:sharedMetalRenderingDevice.device, orientation:.portrait, width:outputWidth, height:outputHeight, timingStyle: .videoFrame(timestamp: Timestamp(currentTime)))
-                    
+                    let outputTexture = Texture(device:sharedMetalRenderingDevice.device, orientation:.portrait, width:outputWidth, height:outputHeight, timingStyle: .videoFrame(timestamp: Timestamp(currentTime)))!
```

Comment:
question:  Is it safe to use force unwrap here?

Reply:
I used force unwrap for the use cases that I didn't find in PicCollage, the purpose is to limit the unrequired changes in GPUImage3.
The only case for the nil result here is the case that it used the fetalError at first, therefore, even if I missed some use case in PicCollage and forced unwrap the value, it won't increase the crash rate.

Reply:
Also, if you still have concern, we can discuss this later on during the code review meeting